src/components/shared/DataTypeSelect.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <traction-select
+    id="data-type"
+    :value="value"
+    :options="dataTypes"
+    @input="handleInput"
+  ></traction-select>
+</template>
+
+<script>
+import useSWRV from 'swrv'
```

Comment:
Ideally I'd like to pull a lot of the swrv contents out into a separate file, or, if not possible a mixin.